code/modules/projectiles/guns/ballistic/bows/_bow.dm

```diff
@@ -44,6 +44,7 @@
 		balloon_alert(user, "no arrow nocked!")
 	else
 		balloon_alert(user, "[drawn ? "string released" : "string drawn"]")
+		playsound(src, 'modular_skyrat/modules/tribal_extended/sound/sound_weapons_bowdraw.ogg', 75, 0) //SKYRAT EDIT - Bow Consolidation
```

Comment:
Well, it could be easily be modular with just checking for `chambered` and calling `..()`, AFAIK, but I don't think that's critical?